samples/nextjs/scripts/generate-config.ts

```diff
@@ -18,6 +18,7 @@ export function generateConfig(configOverrides?: { [key: string]: string }): voi
     sitecoreApiKey: 'no-api-key-set',
     sitecoreApiHost: '',
     jssAppName: 'Unknown',
+    trackingEnabled: 'true',
```

Comment:
Was using a string type here intentional?

Reply:
yes, as generate-config will output it as the string anyway.